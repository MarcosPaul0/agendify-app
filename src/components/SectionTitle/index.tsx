import { Text } from 'react-native';
import { ISectionTitleProps } from './interfaces/sectionTitleProps.interface';

export function SectionTitle({ title, margin = 'mb-5' }: ISectionTitleProps) {
  return (
    <Text
      className={`
        w-full ${margin}
        text-xl font-bold text-BLUE_500
      `}
    >
      {title}
    </Text>
  );
}
