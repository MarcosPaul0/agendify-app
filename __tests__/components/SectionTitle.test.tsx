import { render } from '@testing-library/react-native';
import { SectionTitle } from '@components/SectionTitle';

describe('Section Title', () => {
  it('Match with snapshot', async () => {
    const tree = render(<SectionTitle title="fake-title" />);

    expect(tree).toMatchSnapshot();
  });
});
