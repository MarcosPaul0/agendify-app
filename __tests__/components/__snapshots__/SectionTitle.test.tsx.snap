// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Title Match with snapshot 1`] = `
<Text
  style={
    [
      {
        "width": "100%",
      },
      {
        "marginBottom": 20,
      },
      {
        "fontSize": 20,
        "lineHeight": 28,
      },
      {
        "fontWeight": "700",
      },
      {
        "color": "#3763A4",
      },
    ]
  }
>
  fake-title
</Text>
`;
